FixMultiplication.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+ public class FixMultiplication
+ {
+     
+    public static int isMissing(int t,int T)
+    {
+           int count=0;
+   int missing_digit = -1;
+   int[] ht = new int[10];
+   while(t!=0)
+   {
+	ht[t%10]++;
+	t/=10;
+   }
+   while(T!=0)
+   {
+	ht[T%10]--;
+	T/=10;
+   }
+   for(int i=0;i<=9;i++)
+   {
+	 if(ht[i]==1)
+	 {
+		  count++;
+		  missing_digit = i;
+	 }
+   }
+   if(count!=1)
+	   missing_digit = -1;
+   return missing_digit;
+    }
+       int FindDigit(string inp)
```

Comment:
This method should be public, as it is being accessed from Program.cs
Also, invoke this method from the Main method in Program.cs file.